check_process

```diff
@@ -6,31 +6,28 @@
 ;; Test complet
 	; Manifest
 		domain="domain.tld"	(DOMAIN)
-		path="/"	(PATH)	
+		path="/path"	(PATH)
+		nextclouddomain="domain.tld"
 		is_public=1	(PUBLIC|public=1|private=0)
-#		password="pass"
-#		nextclouddomain="domain.tld"
-		port="9980"	(PORT)
 	; Checks
 		pkg_linter=1
-		setup_sub_dir=0
+		setup_sub_dir=1
 		setup_root=1
 		setup_nourl=0
-		setup_private=0
+		setup_private=1
 		setup_public=1
 		upgrade=1
```

Comment:
Could you add a new line with: `upgrade=1          from_commit==c7d09d7f15a01e85e0354561d737523e1cb18749`.
(and keep the line `upgrade=1` as well).

With this line, the package_check will test the upgrade from the old version to this new one.

Reply:
It's done :)

Reply:
but that was: `upgrade=1 from_commit=c7d09d7f15a01e85e0354561d737523e1cb18749`

Reply:
Oups indeed